cosmwasm/utils.js

```diff
@@ -420,6 +420,28 @@ const makeMultisigProverInstantiateMsg = (config, chainName) => {
     };
 };
 
+const makeAxelarnetGatewayInstantiateMsg = (config, chainName) => {
+    const {
+        axelar: { contracts },
+    } = config;
+    const chainConfig = getChainConfig(config, chainName);
+
+    const { axelarId: chainId } = chainConfig;
```

Comment:
let's avoid introducing multiple terms for the same value (chainId means something different for external chains). Instead of destructuring all fields, use it as `chainConfig.axelarId`. It's more readable.